src/test/end-to-end/features/claimDetails/claimDetailsTest.js

```diff
@@ -0,0 +1,23 @@
+const { doNotHaveToCompleteCard, stepsToMakingYourClaim, claimDetails } = require('../../helpers/caseHelper');
+const commonFlow = require('../../helpers/commonFlow.js');
+const { I } = inject();
+
+Feature('Enter claim details for a claim');
+
+Scenario('Navigate Claim Details - Describe What happenned to you', async () => {
+  await commonFlow.createSingleMyselfCase();
```

Comment:
I was thinking for these tests we navigate to the page directly or to the task list page. The e2e steps will cover the e2e journey. https://github.com/hmcts/et-sya-frontend/blob/master/src/test/end-to-end/features/e2e_Scenarios/submitClaimForSinglePerson.js

Reply:
This has been updated as requested...